### DISCUSSION TITLE
esse projeto está ativo?

### DISCUSSION THREAD

**Discussion by ghost** [po-ui/po-angular#1588] [Tire suas dúvidas] [answered]
Olá a todos.

esse projeto está ativo?

*gostaria de contribuir com algumas ideias*

**Comment by alinelariguet** [comment, marked as answer]
Bom dia @codehangen está ativo sim!

**Reply by ghost** [reply]
Oi alinelariguet.

> Bom dia @codehangen está ativo sim!

fico feliz com sua resposta, então vou ajudar e contribuir da melhor forma possível ^^
